pkg/helper/client.go

```diff
@@ -15,15 +16,34 @@ import (
 
 const nmstateCommand = "nmstatectl"
 
-func nmstatectl(arguments ...string) (string, error) {
-	cmd := exec.Command(nmstateCommand, arguments...)
-	var outb, errb bytes.Buffer
-	cmd.Stdout = &outb
-	cmd.Stderr = &errb
+func show(arguments ...string) (string, error) {
+	cmd := exec.Command(nmstateCommand, "show")
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
 	if err := cmd.Run(); err != nil {
-		return "", fmt.Errorf("Failed to execute nmstatectl show: '%v'", err)
+		return "", fmt.Errorf("Failed to execute nmstatectl show: '%v', %s", err, stderr.String())
 	}
-	return outb.String(), nil
+	return stdout.String(), nil
+}
+
+func set(state string) error {
```

Comment:
I think we should capture stdout (or whatever contains log) output and log it in the pod. I think it is a valuable information, especially now when we may face a lot of issues with config.

Reply:
Done